docs/strat-lib/technical-references/code/strategies/interfaces/ILiquidityProvider.md

```diff
@@ -0,0 +1,43 @@
+## ILiquidityProvider
```

Comment:
Do we want to add Code generated doc, in the same PR?

Reply:
It will have broken links unless we add it.